G1ANT.Addon.UI/Helpers/VirtualKeyShortConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Windows.Forms;
+using FlaUI.Core.WindowsAPI;
+using G1ANT.Language;
+
+namespace G1ANT.Addon.UI.Helpers
+{
+    public static class VirtualKeyShortConverter
+    {
+        public static VirtualKeyShort GetVirtualKeys(string keyValue)
+        {
+            var value = KeyStr.ToModifierKey(keyValue);
+            if (value == ModifierKeys.None)
+            {
+                value = KeyStr.ToModifierKey(keyValue + "+");
+                if (value == ModifierKeys.None)
+                {
+                    var singleKey = KeyStr.ToSingleKey(keyValue).ToString();
+                    return (VirtualKeyShort) (Keys) Enum.Parse(typeof(Keys), singleKey);
+                }
+            }
+
+            var keys = (Keys) Enum.Parse(typeof(Keys), value.ToString());
+            return (VirtualKeyShort) Enum.Parse(typeof(VirtualKeyShort), keys.ToString(), true);
```

Comment:
above you convert the enum (Keys -> VirtualKeyShort) by value, here by name, if the conversion by value is accurate, use it here

Reply:
It is not, that's why I did it

Reply:
Good catch!